packages/components/src/components/tag/_tag.scss

```diff
@@ -171,6 +171,10 @@
     background-color: transparent;
     border-radius: 50%;
     cursor: pointer;
+
+    &:disabled {
```

Comment:
Style-nit: would be great to un-nest these styles to help with maintenance down the line 👀

Reply:
Done ✔️
Also noticed that it's probably better to target the close-icon element in a `.bx--tag--disabled` in general instead of using the `:disabled` attribute.
Let me know what you think!